ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/isis_system_id_format.schema.yml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+# TODO: AVD5.0 flip default to underlay_loopback
+type: dict
+keys:
+  isis_system_id_format:
+    display_name: ISIS Default IS System-ID format
+    description: Configures source for the system-id within the ISIS net id. If `underlay_loopback` is selected then all node `isis_system_id_prefix` settings will be ignored.
```

Comment:
Please add some information about how the ID is generated for the `node_id` option.